izpack-compiler/src/main/java/com/izforge/izpack/compiler/packager/impl/PackagerBase.java

```diff
@@ -332,48 +349,48 @@ public Map<String, List<DynamicVariable>> getDynamicVariables()
      * @return the dynamic conditions
      */
     @Override
-    public List<DynamicInstallerRequirementValidator> getDynamicInstallerRequirements()
+    public final List<DynamicInstallerRequirementValidator> getDynamicInstallerRequirements()
     {
         return dynamicInstallerRequirements;
     }
 
     @Override
-    public void addInstallerRequirements(List<InstallerRequirement> conditions)
+    public final void addInstallerRequirements(List<InstallerRequirement> conditions)
     {
         this.installerRequirements = conditions;
     }
 
     @Override
-    public void createInstaller() throws Exception
+    public final void createInstaller() throws Exception
     {
+        System.err.println("******* start createInstaller");
```

Comment:
Why on System.err and not use logger? Same on line 378.

Reply:
Forgot to remove after locating the bug...